modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/extras/MatchOnlyTextFieldMapperTests.java

```diff
@@ -265,7 +265,7 @@ protected IngestScriptSupport ingestScriptSupport() {
     }
 
     @Override
-    protected Function<Object, Object> loadBlockExpected(MapperService mapper, String loaderFieldName) {
+    protected Function<Object, Object> loadBlockExpected() {
```

Comment:
I'm not sure I follow? I just reverted this method to the version before Luigi's PR. Neither his change nor mine changed the return type.

Reply:
Just saying that it _might_ be helpful to expose the type in the method signature `protected Function<Object, BytesRef>`.
it's used in testing so it might not matter in the end.